apps/generic_terminal/unix_shell.talon

```diff
@@ -0,0 +1,26 @@
+tag: terminal
+tag: user.generic_unix_shell
+os: mac
+os: linux
+-
+# Generic Shell Commands. (General convension is to use the command name in "man" when possible)
+
+^change (directory|dir)$: "cd "
+^home (directory|dir)$: "~/"
+^copy$: "cp "
+^make dir$: "mkdir "
+^cancel$: key(ctrl-c)
+^(move|rename) command: "mv "
+^list directory$: "ls "
+^list current directory$: user.terminal_list_directories()
+^grep$: "grep "
+^edit$: "vi " # Can be replaced with your favorite editor
+^less$: "less "
+^remove$: "rm "
+^teemux$: "tmux "
+^netstat$: "netstat "
+^which$: "which "
+^touch$: "touch "
```

Comment:
This will conflict with the mouse `touch` command.

Reply:
good point. I am suggesting to disambiguate this by adding the optional "command" to it:
1) ^touch [command]$: ...
or prefix with "shell"
2) ^[shell] touch$:
What do you think?
We can establish those rules here so they can be followed going on forward